package.js

```diff
@@ -0,0 +1,36 @@
+// package metadata file for Meteor.js
+var packageName = 'urigo:angulartics'; // https://atmospherejs.com/urigo/angulartics
+var where = 'client'; // where to install: 'client' or 'server'. For both, pass nothing.
+var version = '0.17.2_1';
+
+Package.describe({
+  name: packageName,
+  version: version,
+  summary: 'angulartics (official): Analytics for AngularJS applications',
+  git: 'git@github.com:luisfarzati/angulartics.git'
+});
+
+Package.onUse(function(api) {
+  api.versionsFrom(['METEOR@0.9.0', 'METEOR@1.0']);
+
+  api.use('angular:angular@1.1.5', where);
```

Comment:
Once we package for npm both Angulartics core and all its plugins, shouldn't be as easy as doing what you are doing here for requiring Angular?